### ISSUE TITLE
RDF/JSON serialisation doesn't work for certain resources

### ISSUE THREAD

**Issue by townxelliot** [bbcarchdev/quilt#55]
While writing some code to search RES and fetch resources, I spotted a few issues with the RDF/JSON serialiser. The URIs below return Turtle perfectly happily (of the ones I've tested), but return 406 errors when the resource is requested with "Accept: application/json". Similarly, a GET on the .rj URI returns a 406.

Is this intended behaviour? Should I expect that some resources won't return JSON?

http://acropolis.org.uk/0fb0512c02fa489389a9dfe9fa9da6f6#id
http://acropolis.org.uk/14e52c994cd44f4589429af5d8fb58f6#id
http://acropolis.org.uk/206dbeccb627458194dd60e8c3d6540e#id
http://acropolis.org.uk/2927f0325fe94e8e9082ba4699230b49#id
http://acropolis.org.uk/38ca0f310874487aaf56405b5b2938eb#id
http://acropolis.org.uk/41c12ea886fb4470bb0be985abd48dd3#id
http://acropolis.org.uk/597f064dd27a41e1868acd2e3c47e31f#id
http://acropolis.org.uk/5cf44aa9ea174e63aef9c9fcf3ed1072#id
http://acropolis.org.uk/5dfddde913424a118a4de54c8f449f7f#id
http://acropolis.org.uk/6c1e6427f4e8484db6e3a98a814fbd77#id
http://acropolis.org.uk/99815ce258ea4d87b07891d6ca70d991#id
http://acropolis.org.uk/b7270fad8cdc424c9a9bb9fd8a4ec11c#id
http://acropolis.org.uk/d07af40b3d344e22968f84be340247fb#id
http://acropolis.org.uk/f0afc9b094b142ac92bd9f6ab76f3347#id

**Comment by nevali** [comment]
Nope, all resources should provide RDF/JSON, in fact the model-population and serialisation are dealt with by two completely separate projects (spindle versus quilt), and so there shouldn't be anything (besides a bug) which causes one resource to be serialisable and another not
